core/src/main/java/org/sonarsource/sonarlint/core/WsHelperImpl.java

```diff
@@ -31,11 +34,15 @@
 import org.sonarsource.sonarlint.core.container.connected.validate.DefaultValidationResult;
 import org.sonarsource.sonarlint.core.container.connected.validate.PluginVersionChecker;
 import org.sonarsource.sonarlint.core.container.connected.validate.ServerVersionAndStatusChecker;
+import org.sonarsource.sonarlint.core.telemetry.TelemetryPayload;
+import org.sonarsource.sonarlint.core.util.ws.HttpConnector;
+import org.sonarsource.sonarlint.core.util.ws.PostRequest;
 import org.sonarsource.sonarlint.core.util.ws.WsResponse;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
 public class WsHelperImpl implements WsHelper {
+  private static String TELEMETRY_ENDPOINT = "";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this field "TELEMETRY_ENDPOINT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3008)